Libraries/LibWeb/CSS/StyleValues/CalculatedStyleValue.cpp

```diff
@@ -3217,19 +3217,19 @@ NonnullRefPtr<CalculationNode const> simplify_a_calculation_tree(CalculationNode
                 [&](Angle const& angle) -> RefPtr<NumericCalculationNode const> {
                     VERIFY(context.percentages_resolve_as == ValueType::Angle);
                     if (angle.type() == Angle::Type::Deg)
-                        return nullptr;
+                        return NumericCalculationNode::create(angle.percentage_of(*percentage), context);
```

Comment:
I have no idea why I had `nullptr` here or why nobody caught that for so long. :laughing:

Reply:
Yeah, I was a bit surprised when I found this. Maybe it is because of the else branch (step 1.2), which is a similar-looking `visit()` where `nullptr` is the correct thing for canonical units.